internal/pkg/util/uri/uri.go

```diff
@@ -19,6 +19,8 @@ const (
 	HTTP = "http"
 	// HTTPS is the keyword for https ref
 	HTTPS = "https"
+	// Oras is the keyword for an oras ref
```

Comment:
I realize you're following the pattern already set out, but this comment (and the others) don't add any clarity IMO.

Reply:
I agree that it is redundant, but since the constant names don't give any insight for their use I think this is ok to state their purpose